pxblue/react-native-cli-templates
# Update RN templates

#### Changes proposed in this Pull Request:
- Use the latest dependencies
- Update the folder structure of the templates
- Simplify the linking scrips
- Ensure that the default test passes in every template

>TODO: Need to update the workflow dependency to 3.0.0 once it is published (see https://github.com/pxblue/react-native-workflows/pull/48)
